include/taskolib/Sequence.h

```diff
@@ -265,6 +272,13 @@ class Sequence
      */
     void execute(Context& context, CommChannel* comm_channel);
 
+    /**
+     * Get the step setup script object.
```

Comment:
You seem to be very strict 😉 . Removed. On the other hand the meaning of the brief description and the return is now duplicated. But I will leave it as it is. Maybe it can be later simplified to `/// Get the step setup script.`.

Reply:
> Maybe it can be later simplified to `/// Get the step setup script.`.

:+1: